apps/trading/components/accounts-container/accounts-container.tsx

```diff
@@ -75,3 +85,31 @@ export const AccountsContainer = ({
     </div>
   );
 };
+
+type Store = {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  filterModel?: { [key: string]: any };
+  columnState?: ColumnState[];
+};
+
+const useAccountStore = create<{
+  gridStore: Store;
+  update: (gridStore: Store) => void;
+}>()(
+  persist(
+    subscribeWithSelector((set) => ({
```

Comment:
`subscribeWithSelector` will not give any performance gain if selector returns new array each time.
`Object.is([store.gridStore, store.update], [store.gridStore, store.update])` is always false 

To make it work effectively we can split usage in two hook calls.

```
const gridStore = useAccountStore((store) => store.gridStore);
const update =   useAccountStore((store) => store.update);
```

Reply:
Actually not sure why I'm using subscribeWithSelector. Its not required. Will remove

Reply:
Updated to use different selectors